driver/camera.c

```diff
@@ -852,6 +852,18 @@ esp_err_t camera_probe(const camera_config_t* config, camera_model_t* out_camera
         vTaskDelay(10 / portTICK_PERIOD_MS);
         gpio_set_level(config->pin_reset, 1);
         vTaskDelay(10 / portTICK_PERIOD_MS);
+    } else if(config->pin_pwdn >= 0) {
```

Comment:
Sure, but why reset twice? If you toggle the power I don't see any reason to reset again (or the other way around)

Reply:
because in reality OV2640 has both PWDN and RST pins. They are not the same and both need to be taken care of if defined. PWDN must be LOW and RST HIGH for the camera to boot. (surely not in AI-Thinker's board though... they seem to like doing things backwards...)

Reply:
Ok, I didn't consider that. I'll push an update.